### PR TITLE
removed matrix job for black

### LINKED ISSUE #9: No longer need two CI jobs for black
# Description
Turns out you can call `black` with multiple `target-version` flags:
```
--target-version py37 --target-version py38
```
so we no longer need two separate CI jobs.

The equivalent option in `pyproject.toml` is
```
target-version = ["py37", "py38"]
```

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
Closes #9.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

### CODE REVIEW

**Comment by bernardopacini** [comment]
This looks good to me. Presumably we cannot do the same for flake8, right?

**Review by bernardopacini — APPROVED** [review]

File: azure/azure_style.yaml [hunk 1]
```diff
@@ -44,17 +44,11 @@ jobs:
...
-    matrix:
-      'python37':
-        PYTHON_VERSION: '3.7'
-      'python38':
-        PYTHON_VERSION: '3.8'
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: $(PYTHON_VERSION)
+      versionSpec: 3.8
```
Comment (bernardopacini) [inline azure/azure_style.yaml:50]: In a few other spots we have this as:

```
  - task: UsePythonVersion@0
    inputs:
      versionSpec: '3.8'
```

with `` around 3.8. I am not sure if we want that for consistency here too.

Comment (nwu63) [reply azure/azure_style.yaml:50]: Oh yeah, I am not sure whether those are needed but I'll add them in.

**Comment by nwu63** [comment]
> This looks good to me. Presumably we cannot do the same for flake8, right?

I don't believe so, no. Actually `black`'s documentation was not terribly clear and I had to dig into GitHub issues to see that you can pass multiple versions to black.

File: azure/azure_style.yaml (hunk 1 above)
Comment (bernardopacini) [reply azure/azure_style.yaml]: Got it — sounds good. Thanks for fixing this!

File: azure/azure_style.yaml [hunk 2]
```diff
@@ -44,17 +44,11 @@ jobs:
...
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: $(PYTHON_VERSION)
+      versionSpec: '3.8'
   - script: |
       cd ${{ parameters.REPO_NAME }}
       pip install black==20.8b1
-      python -m black . --check --diff -l 120
+      black . --check --diff -l 120 --target-version py37 --target-version py38
```
Comment (sseraj) [inline azure/azure_style.yaml:54]: This didn't work without `python -m` when we tried it, but I think that was before we had `UsePythonVersion@0` working properly. Maybe it's worth a shot?

Comment (nwu63) [reply azure/azure_style.yaml:54]: Yeah I'm like 95% sure that this works fine.

Comment (sseraj) [reply azure/azure_style.yaml:54]: Okay, let's try it

**Review by sseraj — APPROVED** [review]

**Comment by sseraj** [comment]
I manually ran pySpline and it works

**Pull request merged**